modules/openapi-generator/src/main/resources/typescript/http/http.mustache

```diff
@@ -98,8 +98,12 @@ class URLParse {
 {{/platforms}}
 
 export class HttpException extends Error {
-    public constructor(msg: string) {
+    code: number | undefined;
+    public constructor(msg: string, code?: number) {
```

Comment:
this can be simplified to
```suggestion
    public constructor(msg: string, public code?: number) {
```
then no assignment `this.code = ...` is necessary

Reply:
thanks for the suggestion. I've implemented it